<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButtonAnimation">
     <property name="text">
      <string>animation</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButtonQuitter">
     <property name="text">
      <string>quitter</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGraphicsView" name="graphicsView">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>250</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>250</width>
           <height>500</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGraphicsView" name="graphicsView4"/>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView3"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>833</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
